.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
?

Reply:
this is a hint for the 'nvm' binary ... with this I can go into this repo and execute `nvm use` ... and the version given in this .nvmrc will automatically installed and enabled.